### PR TITLE
Tweak API docstrings/comments

### PR DESCRIPTION
**Pull request by aekiss**

### CODE REVIEW

File: regional_mom6/regional_mom6.py [hunk 1]
```diff
@@ -1713,9 +1717,8 @@ def __init__(
 
     def rectangular_brushcut(self):
         """
-        Cut out and interpolates tracers. This method assumes that the boundary
-        is a simple Northern, Southern, Eastern, or Western boundary. Cuts out
-        and interpolates tracers.
+        Cut out and interpolate tracers. This method assumes that the boundary
+        is a simple Northern, Southern, Eastern, or Western boundary.
         """
         if self.orientation == "north":
```
Comment (aekiss) [inline regional_mom6/regional_mom6.py:1723]: Should this sequence of `if` statements be `elif'`s, with a final `else` to abort on unknown orientation?

(Or have an assert that orientation is one of allowed options?

Comment (aekiss) [reply regional_mom6/regional_mom6.py:1723]: Similarly with `if self.grid` sequence below

**Review by navidcy — APPROVED** [review]
great work!
lgtm!

File: regional_mom6/regional_mom6.py (hunk 1 above)
Comment (navidcy) [reply regional_mom6/regional_mom6.py:1730]: I like this suggestion.
@ashjbarnes, what do you think?

**Comment by navidcy** [comment]
I took the liberty to push a few tweaks...

File: regional_mom6/regional_mom6.py (hunk 1 above)
Comment (navidcy) [reply regional_mom6/regional_mom6.py:1742]: done via [81813aa](https://github.com/COSIMA/regional-mom6/pull/158/commits/81813aa177bebe56871bafc3228c84424662b7e3) in a slightly different way than what @aekiss suggested

**Pull request merged**